ldpd/lde_lib.c

```diff
@@ -919,6 +919,7 @@ lde_gc_timer(struct thread *thread)
 		    !RB_EMPTY(lde_map_head, &fn->upstream))
 			continue;
 
+		lde_free_label (fn->local_label);
```

Comment:
Looks to me as if this can be ```NO_LABEL```, so that would be a good test to have?
Also, a style issue: no space before the open-parenthesis please.

Reply:
Thank your for your review comments. 
I will change it like,
     if (fn->local_label != NO_LABEL)
            lde_free_label(fn->local_label);